lib/autoload/coursemology.rb

```diff
@@ -0,0 +1,7 @@
+module Coursemology
+  class ComponentNotFoundError < StandardError
```

Comment:
How about `Componentize::ComponentNotFound`, that is `lib/autoload/componentize.rb`